packages/mask/src/plugins/Avatar/SNSAdaptor/NFTAvatarClip.tsx

```diff
@@ -145,9 +148,13 @@ export function NFTAvatarClip(props: NFTAvatarClipProps) {
     const { id = uuid(), width, height, viewBoxHeight = ViewBoxHeight, viewBoxWidth = ViewBoxWidth, screenName } = props
     const classes = useStylesExtends(useStyles(), props)
     const { loading, value: avatarMetadata } = useNFTContainerAtTwitter(screenName ?? '')
+    const account = useAccount()
     const { value = { amount: '0', symbol: 'ETH', name: '', slug: '' }, loading: loadingNFT } = useNFT(
```

Comment:
Why the symbol of an NFT token is `ETH`?

Reply:
this is twitter nft

Reply:
this is twitter nft